packages/flame/lib/src/text/styles/inline_text_style.dart

```diff
@@ -84,10 +87,15 @@ class InlineTextStyle extends FlameTextStyle {
   }
 
   TextStyle asTextStyle() {
+    final fontSize = this.fontSize;
+    if (fontSize == null) {
+      throw Exception('fontSize must be set');
```

Comment:
Isn't an assertion enough here?

Reply:
I plan on working on standardizing asserts, exceptions and tests thereof on a followup, so I am not making any particular choice in this PR.

Reply:
But you did make the choice of casting an unspecified exception, it didn't have that before right? Which is like the worst of both worlds 😅
Or are you doing the refactoring like today? Then I guess it is fine.